sdk/database/dbplugin/database.proto

```diff
@@ -78,6 +79,26 @@ message RotateRootCredentialsResponse {
 
 message Empty {}
 
+message GenerateCredentialsResponse {
+  string password = 1;
+}
+
+message StaticUserConfig{
+  string Username = 1;
```

Comment:
Can you normalize on lowercase variable names?

Reply:
Yes, done in 2ca4849e3